<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLabel" name="textValIterations">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="valSeuil">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>50.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textSeuil">
       <property name="text">
        <string>Valeur du seuil</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="seuilButton">
       <property name="text">
        <string>Appliquer seuil</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="sliderHaar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="saveFinButton">
       <property name="text">
        <string>Save image compressée</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="saveAffButton">
       <property name="text">
        <string>Save image affichée</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choix de l'itérations de Haar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Ouvrir</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>170</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Taux de compression (%) : </string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="valTaux">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="labelImage">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>browseInputPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHaar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>haarNb(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAffButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>saveAffichee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveFinButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>saveFinale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seuilButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>unhaar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHaar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>textValIterations</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseInputPath()</slot>
  <slot>haarNb(int)</slot>
  <slot>saveAffichee()</slot>
  <slot>saveFinale()</slot>
  <slot>unhaar()</slot>
 </slots>
</ui>
